platform-emf/src/test/java/com/softicar/platform/emf/form/EmfFormTest.java

```diff
@@ -210,14 +217,53 @@ public void testDirectEditOfEntityWithoutClosing() {
 		frame.assertIsChangedOnRefreshBus(entity);
 	}
 
+	@Test
+	public void testViewScopeOfEntityWithoutPermission() {
+
+		// insert entity and show form
+		showForm(insertScopedTestObject(scopeObject, SOME_NAME, SOME_DAY));
+
+		// assert that no view scope action is available
+		findNodes(new EmfCommonActionMarker(EmfFormViewScopeAction.class)).assertNone();
+	}
+
+	@Test
+	public void testViewScopeOfEntityWithPermission() {
+
+		// add current user to view permission
+		scopeObject.addAuthorizedUser(user);
+
+		// insert entity and show form
+		showForm(insertScopedTestObject(scopeObject, SOME_NAME, SOME_DAY));
+		clickButton(new EmfCommonActionMarker(EmfFormViewScopeAction.class));
+
+		// assert that the scope popup is displayed
+		findFormPopup(EmfTestObject.class).assertDisplayed();
+	}
+
```

Comment:
I think we need an additional test method to ensure that there is no button (and that we don't crash) if the table has no scope field.